test/unit/index.js

```diff
@@ -1,269 +1,12 @@
 /*!
- * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2019-2023 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
+import {createRequire} from 'node:module';
+import {getMockHmac} from '../MockHmac.common.js';
+import {test} from './common.js';
+const require = createRequire(import.meta.url);
+const crypto = require('isomorphic-webcrypto');
```

Comment:
We can actually remove `isomorphic-webcrypto` as a test dependency and replace it with what the `crypto.js` and `crypto-browser.js` files do here:

https://github.com/digitalbazaar/minimal-cipher/tree/main/lib

It looks like there's already an `index.js` and an `index.browser.js` here -- so you can just do the proper import of `crypto` in each of those files ... and maybe the `index.browser.js` one already works fine. So it may just be a change here to do:

```js
import {webcrypto as crypto} from 'node:crypto';
```

Or some variant of that that may take two lines (an import and a destructure).

Reply:
Updated [586d9a5](https://github.com/digitalbazaar/minimal-jwt/pull/4/commits/586d9a5e507bd422242426435013398dc40aa8d8)